api/lib/infrastructure/repositories/campaign-participant-repository.js

```diff
@@ -140,6 +141,9 @@ async function _getOrganizationLearner(campaignId, userId, domainTransaction) {
       function () {
         this.on('campaign-participations.organizationLearnerId', 'organization-learners.id');
         this.on('campaign-participations.campaignId', 'campaigns.id');
+        this.on('campaign-participations.deletedAt', knex.raw('IS'), knex.raw('NULL'));
+        this.on('campaign-participations.isImproved', knex.raw('IS'), knex.raw('false'));
```

Comment:
```suggestion
        this.on('campaign-participations.isImproved', false);
```

Reply:
ça marchait pas mais j'ai un équivalent :         this.on('campaign-participations.isImproved', knex.raw('false'));